modules/cert-manager-proxy-support.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-proxy-support_{context}"]
+= TODO: Title
+
+TODO: intro
+
+////
+If a cluster wide egress proxy is configured on the OpenShift cluster, OLM automatically update all the operators' deployments with HTTP_PROXY, HTTPS_PROXY, NO_PROXY environment variables.
+Those variables are then propagated down to the cert-manager (operand) controllers by the cert manager operator.
+Trusted Certificate Authority
+Running operator
+
+Follow the instructions below to let Cert Manager Operator trust a custom Certificate Authority (CA). The operator's OLM subscription has to be already created.
+////
+
+.Prerequisites
+
+* TODO
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a config map in the `cert-manager` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create configmap trusted-ca -n cert-manager
+----
+
+. Inject the CA bundle that is trusted by {product-title} into the config map by running the following command:
++
+[source,terminal]
+----
+$ oc label cm trusted-ca config.openshift.io/inject-trusted-cabundle=true -n cert-manager
+----
+
+. Update the deployment for the {cert-manager-operator} to use the config map by running the following command:
++
+[source,terminal]
+----
+$ oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
+----
++
+////
+This failed for me:
+[ahoffer@ahoffer march2] $ oc -n cert-manager-operator patch subscription cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}}'
+Error from server (NotFound): subscriptions.operators.coreos.com "cert-manager-operator" not found
+////
+
+. TODO: Wait for rollout. Command to check? Or say wait <x> minutes?
```

Comment:
@thejasn @swghosh This says to wait for rollout. Is there an estimated number of minutes we should say to wait, or some status that they can check to know when the rollout is complete?

Reply:
```
$> oc rollout status deployment/cert-manager-operator-controller-manager -n cert-manager-operator && \
oc rollout status deployment/cert-manager -n cert-manager && \
oc rollout status deployment/cert-manager-webhook -n cert-manager && \
oc rollout status deployment/cert-manager-cainjector -n cert-manager
```

Reply:
Perfect thanks!